### PR TITLE
Milestone 2 : Added Works Sections

### PR DESCRIPTION
**Pull request by AngomRanjan**
## Profile

In this milestone :
-  Added Work Section
- Added About Me Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,15 +23,100 @@
...
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-2">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder2"><img src="images/img-project2.png" alt="Project-2 Snapshoot"></div>
+      <div class="info-blocks" id="info-block2">
+        <h1 class="project-titles" id="project-title2">Multi-Post Stories</h1>
```
Comment (Deepakdanger) [inline index.html]: - [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.

File: index.html [hunk 2]
```diff
@@ -23,15 +23,100 @@
...
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-3">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder3"><img src="images/img-project3.png" alt="Project-3 Snapshoot"></div>
+      <div class="info-blocks" id="info-block3">
+        <h1 class="project-titles" id="project-title3">Tonic</h1>
```
Comment (Deepakdanger) [inline index.html]: - [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.

File: index.html [hunk 3]
```diff
@@ -23,15 +23,100 @@
...
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-4">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder4"><img src="images/img-project4.png" alt="Project-4 Snapshoot"></div>
+      <div class="info-blocks" id="info-block4">
+        <h1 class="project-titles" id="project-title4">Multi-Post Stories</h1>
```
Comment (Deepakdanger) [inline index.html]: - [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.

File: index.html [hunk 4]
```diff
@@ -23,15 +23,100 @@
...
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section class="grid-container" id="about-myself">
+    <div class="my-info" id="personal-info">
+      <h1 class="head1" id="about">About Myself</h1>
```
Comment (Deepakdanger) [inline index.html]: - [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @AngomRanjan,

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
- [ ] All cards in the `work` section have misplaced the `See project` button kIndly fix it.
Screenshot from 2022-04-23 23-35-06
 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (AngomRanjan) [reply index.html]: Fixed as suggested

**Review by vic778 — APPROVED** [review]
Hi @AngomRanjan  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**